src/styles/style.css

```diff
@@ -0,0 +1,159 @@
+.body {
+  margin: 0;
+
+  --border-color: rgb(240, 7, 228);
+
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  width: 199px;
+  height: 100%;
+  border: 1px solid var(--border-color);
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.card__image {
+  box-sizing: border-box;
+  background-image: url(/src/images/imac.jpeg);
+  background-size: cover;
+  width: 160px;
+  height: 134px;
+  margin: 33px 0 0 20px;
```

Comment:
Avoid using fixed margin values. It's better to use padding within the parent element or consider using a more responsive unit like `em`, `rem`, or percentages.